app/scripts/models/auth_brokers/base.js

```diff
@@ -31,7 +31,13 @@ define([
       var self = this;
       return p()
         .then(function () {
-          self._isForceAuth = self._isForceAuthUrl();
+          self._isForceAuth = self._isForceAuthUrl() ||
+                              self._isSignInWithForceEmail();
+
+          // The `/force_auth` screen expects `email`, not `forceEmail`
+          if (self._isSignInWithForceEmail()) {
+            self.relier.set('email', self.relier.get('forceEmail'));
```

Comment:
> add another param (force_auth?)

I also considered this, and I decided on `force_email` for two reasons:

1. The potential for confusion with [email=blank](https://github.com/mozilla/fxa-content-server/pull/2027). `email=blank&force_auth=true` - does this mean anything?
1. I prefer a single parameter when possible - specifying two parameters when a single parameter works could hinder extension in the future as well as cause confusion `email=<xxx>&force_auth=true` vs `force_email=<xxx>`

I view `force_email` as > `email`. If `force_email` is specified, ignore `email`. Or, error if both are specified. 

I'm not completely sold either way.